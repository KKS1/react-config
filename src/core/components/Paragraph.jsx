import React, { Component } from "react";

const Paragraph = props => {
  const { text } = props.schema ? props.schema : props;
  return <p> {text} </p>;
};

export default Paragraph;
