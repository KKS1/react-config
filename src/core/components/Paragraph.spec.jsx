import React, { Component } from "react";
import Paragraph from "./Paragraph";

describe("Paragraph", () => {
  test("render_Paragraph", () => {
    const props = {
      text: "paragraph content"
    };
    const wrapper = shallow(<Paragraph {...props} />);
    const instance = wrapper.getElement();
    expect(instance).toMatchSnapshot();
  });

  test("render_Paragraph_with_schema", () => {
    const props = {
      schema: { text: "paragraph content" }
    };
    const wrapper = shallow(<Paragraph {...props} />);
    const instance = wrapper.getElement();
    expect(instance).toMatchSnapshot();
  });
});
