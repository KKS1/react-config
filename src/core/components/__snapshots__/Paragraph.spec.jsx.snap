// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph render_Paragraph 1`] = `
<p>
   
  paragraph content
   
</p>
`;

exports[`Paragraph render_Paragraph_with_schema 1`] = `
<p>
   
  paragraph content
   
</p>
`;
